libs/filamat/src/Enums.cpp

```diff
@@ -47,6 +47,8 @@ std::unordered_map<std::string, Property> Enums::mStringToProperty = {
         { "ior",                 Property::IOR },
         { "microThickness",      Property::MICRO_THICKNESS },
         { "bentNormal",          Property::BENT_NORMAL },
+        {"specularFactor",       Property::SPECULAR_FACTOR },
```

Comment:
Let's use a different name here. Like `specularIntensity`.

Reply:
Also nit: watch the formatting (missing space after `{`)

Reply:
have some difficulty to change it to specularIntensity, keep reporting cannot find this variable in all the shaders. Nee to deal with this renaming latter.